libraries/ui-library/src/components/six-file-upload/six-file-upload.scss

```diff
@@ -40,6 +40,19 @@ $margin: 1rem;
     width: 100%;
     transition: background-color $transition;
 
+    &__spinner-container {
+      display: flex;
+      align-items: center;
+
+      six-spinner {
+        margin-right: var(--six-spacing-x-small);
+      }
+    }
+
+    &--disabled &__drop-zone {
+      cursor: not-allowed;
```

Comment:
```suggestion
```
Move this into the selector on line 73. Here it doesn't work. (I should have been more specific 🙃)

Reply:
Fixed in the next commit: 
![image](https://github.com/six-group/six-webcomponents/assets/58786119/554006b5-a42a-408d-b047-d3ab232bce6a)